src/create_helpers.ts

```diff
@@ -1,6 +1,14 @@
 import * as ts from 'typescript';
-import { warn } from './logger';
-import { resolveType, createFunctionParams, createFunctionReturnType, resolveTypeParameters, resolveHeritageClauses } from './type_resolve_helpers';
+import {warn} from './logger';
+import {
+    createFunctionParams,
+    createFunctionReturnType,
+    createTypeLiteral,
+    resolveHeritageClauses,
+    resolveType,
+    resolveTypeParameters
+} from './type_resolve_helpers';
+import {PropTree} from "./PropTree";
```

Comment:
nit: Spaces between braces and named import. Here and above where it was removed.